FAQ.md

```diff
@@ -31,22 +30,34 @@ This is especially problematic for PDB files from X-ray crystallography which ty
 
 Given a PDB file of a hypothetical biomolecular system of interest containing a small molecule, there are several routes available to you for treating the small molecule present:
-- Use a cheminformatics toolkit (see above) to infer bond orders
+- Use a cheminformatics toolkit (see below) to infer bond orders
 - Identify your ligand from a database; e.g. if it is in the Protein Data Bank (PDB), it will be present in the [Ligand Expo](http://ligand-expo.rcsb.org) meaning that it has a database entry and code you can use to look up its putative chemical identity
 - Identify your ligand by name or SMILES string (or similar) from the literature or your collaborators
 
+## What about starting from an XYZ file?
+
+XYZ files generally only contain elements and positions, and are therefore similar in content to PDB files. See the above section "What about starting from a PDB file?" for more information.
+
 ## What do you recommend as a starting point?
 
 For application of SMIRNOFF force fields, we recommend that you begin your work with formats which provide the chemical identity of your small molecule (including formal charge and bond order).
 This means we recommend one of the following or equivalent:
-- A `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a `.mol2` file, but the bond orders will be incorrect)
+- A `.mol`, `.sdf`, or `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a correct-seeming file, but the bond orders will be incorrect)
 - Isomeric SMILES strings for the components of your system
 - InCHI strings for the components of your system
 - Chemical Identity Registry numbers for the components of your system
 - IUPAC names for the components of your system
 
 Essentially, anything which provides the full identity of what you want to simulate (including stereochemistry) should work, though it may require more or less work to get it into an acceptable format.
 
+## I understand the risks and want to perform bond and formal charge inference anyway
+
+If you are unable to provide a molecule in the formats recommended above and want to attempt to infer the bond orders and atomic formal charges, there are tools available elsewhere that can provide guesses for this problem. These tools are not perfect, and the inference problem itself is poorly defined, so you should review each output closely. Some tools we know of include:
+
+- the OpenEye toolkits' `OEPerceiveBondOrders` functionality
+- [MDAnalysis' RDKit converter](https://docs.mdanalysis.org/stable/documentation_pages/converters/RDKit.html?highlight=rdkit#module-MDAnalysis.converters.RDKit) 
```

Comment:
Just linking this leaves out a good bit of information

* That MDAnalysis is capable of gobbling up a ton of different formats
* That the inference happens during the MDAnalysis -> RDKit conversion

such that I'm not sure I'd be able to run with this if I didn't have Lily's code from earlier in my head.

In this case, however, I'm fine with leaving users just a few breadcrumbs instead of hand-holding through our API.

Reply:
```suggestion
- [MDAnalysis' RDKit converter](https://docs.mdanalysis.org/stable/documentation_pages/converters/RDKit.html?highlight=rdkit#module-MDAnalysis.converters.RDKit) , with an [example here](https://github.com/openforcefield/openff-toolkit/issues/1126#issuecomment-969712195)
```